inst/rmarkdown/templates/pihph/resources/template.tex

```diff
@@ -0,0 +1,285 @@
+\titleheader{$if(title-header)$$title-header$$else$$title$$endif$}
+
+$if(default-fonts)$
+\setmainfont{Cambria}
+\setlicensefont{Calibri}
+\setlogofont{Lekton}
+$endif$
+$if(alt-fonts)$
+\setmainfont{Caladea}
+\setlicensefont{Carlito}
+\setlogofont{Lekton}
+$endif$
+$if(main)$
+\setmainfont{$main$}
+$endif$
+$if(licensefont)$
+\setlicensefont{$licensefont$}
+$endif$
+$if(logofont)$
+\setlogofont{$logofont$}
+$endif$
+
+
+% If you do not have access to Cambria, you can use the metric-equivalent Caladea: https://fontlibrary.org/en/font/caladea
+% If you do not have access to Calibri, you can use the metric-equivalent Carlito: https://fontlibrary.org/en/font/carlito
+% Lekton available at https://www.fontsquirrel.com/fonts/lekton
+
+% If any of these fonts are not available on your system, feel free to
+% change them here, but of course the final layout will then differ
+
+\usepackage{polyglossia}                   % or use babel if desired
+\setmainlanguage[variant=british]{english} % or use babel if desired
+\usepackage[autostyle, english=british]{csquotes} % remove the english=british option for double quotes
+\usepackage{$gloss$} % for examples, feel free to use anything else
+
+$if(listings)$
+\usepackage{listings}
+$endif$
+$if(lhs)$
+\lstnewenvironment{code}{\lstset{language=Haskell,basicstyle=\small\ttfamily}}{}
+$endif$
+$if(highlighting-macros)$
+$highlighting-macros$
+$endif$
+$if(verbatim-in-note)$
+\usepackage{fancyvrb}
+\VerbatimFootnotes % allows verbatim text in footnotes
+$endif$
+
+% Some recommended packages
+
+\usepackage{booktabs} % for nice tables
+
+$if(biblatex)$
+% BibLaTeX setup, recommended
+\usepackage[backend=biber,
+       style=langsci-unified, % available on CTAN
+       mincrossrefs=50,
+       maxcitenames=3,
+       maxbibnames=50,
+       useprefix=true,
+       doi=false]{biblatex}
+$for(bibliography)$
+\addbibresource{$bibliography$}
+$endfor$
+% remember to include the .bib extension
+% remember to run biber rather than bibtex if using this option
+\AtEveryCitekey{\ifciteseen{}{\defcounter{maxnames}{99}}}
+$endif$
+$if(natbib)$
+% BibTeX setup, deprecated
+\usepackage{natbib}
+\bibpunct[:]{(}{)}{;}{a}{}{,}
+\setlength\bibsep{0pt}
+$endif$
+
+$for(header-includes)$
+$header-includes$
+$endfor$
+
+\begin{document}
+\setcounter{page}{1}
+\maketitle
+\thispagestyle{empty}
+
+$if(abstract)$
+\begin{abstract}
+\noindent $abstract$
+\end{abstract}
+$endif$
+
+$body$
+
+\commentsinvited % Please do not change this
+
+$if(acknowledgements)$
+\section*{Acknowledgements}
+$acknowledgements$
+$endif$
+
+$if(supplement)$
+\section*{Associated material}
+$supplement$
+$endif$
+
+\section*{Author's contact details}
+\label{sec:auth-cont-deta}
+
+$for(author)$
+$if(author.email)$
+\noindent \textit{$author.name$}
+
+\noindent $author.affiliation$
+
+$if(author.address)$
+
+\noindent $author.address$
+$endif$
+\vspace*{5pt}
+\noindent \textit{$author.email$}
+
+$endif$
+$endfor$
```

Comment:
Currently with this we get only one author. See your Skeleton template. 

This is because second author has not `author.email` so no other field is used. Should the if on `author.email` be only around the part for `author.email` ? 

![image](https://user-images.githubusercontent.com/6791940/103637667-172f0b80-4f4c-11eb-98bf-3cf91ec49739.png)

Reply:
4612e89